.github/workflows/ci.yml

```diff
@@ -72,22 +72,28 @@ jobs:
     name: "Generated: ${{ matrix.script.name }} ${{ matrix.extra_options.project_name }}"
     steps:
       - uses: actions/checkout@v4
+      - run: git config --global user.name github-actions[bot]
+      - run: git config --global user.email 41898282+github-actions[bot]@users.noreply.github.com
       - uses: actions/setup-python@v5
         id: setup-python
         with:
           python-version: "3.11"
+      - run: pipx install copier
+      - run: pipx inject copier jinja2-eval jinja2-env jinja2-time arrow
```

Comment:
How does it work if folks don't have these installed? That seems like a significant change in workflow for users

Reply:
Copier raises an error if not installed. There is no workaround. I suppose that if you are interested in jinja2-time, injecting is inevitable anyway.